src/Lykke.Sdk/WebHostFactory.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using System;
+using System.IO;
+
+namespace Lykke.Sdk
+{
+    /// <summary>
+    /// This class creates IWebHostBuilder which is used in LykkeStarter.
+    /// </summary>
+    public class WebHostFactory : IWebHostFactory
+    {
+        /// <inheritdoc />
+        public IWebHostBuilder CreateWebHostBuilder<TStartup>(Action<WebHostFactoryOptions> optionConfiguration) 
+            where TStartup : class
+        {
+            if (optionConfiguration == null)
```

Comment:
Let's make null - default value for optionConfiguration parameter.
For now in most cases default value of WebHostFactoryOptions created by its c-tor is more than enough

Reply:
Disagree. It's not null when it used inside SDK itslef. Since WebHostBuilder is inteded for tests, it's better to always specify port manually.